judoscale-ruby/lib/judoscale/job_metrics_collector.rb

```diff
@@ -27,7 +27,7 @@ def self.adapter_config
     def initialize
       super
 
-      log_msg = +"#{self.class.adapter_name} enabled"
+      log_msg = "#{self.class.adapter_name} enabled"
```

Comment:
It looks like Ruby 2.x didn't like this change... it appears they consider interpolated strings as frozen, whereas 3+ doesn't: (which is probably why standard/rubocop suggested it?)

```ruby
% chruby 3.3.4

% ruby --enable-frozen-string-literal -e 'puts [RUBY_VERSION, "foo".frozen?, "#{%q(zomg)}".frozen?].inspect'
["3.3.4", true, false]

% chruby 2.7.8

% ruby --enable-frozen-string-literal -e 'puts [RUBY_VERSION, "foo".frozen?, "#{%q(zomg)}".frozen?].inspect'
["2.7.8", true, true]
```

I'm not sure I have a good suggestion that doesn't include a check on RUBY_VERSION to `dup` the string for now... (or always `dup` it and leave a note about when dropping older versions or something)

Reply:
The `RedundantInterpolationUnfreeze` check was literally [added to `standard` two days ago](RedundantInterpolationUnfreeze) and subsequently [released in 1.41.0](https://github.com/standardrb/standard/blob/main/CHANGELOG.md#1410).

Another approach here is to leave our code as it was before this PR, and lock our `lint` GH Action to `gem install standardrb -v '~> 1.40.0'`.